test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Remember to delete al unused code 😯.

In the other hand LGTM, just be careful with that. 
Next time I will request changes.

Reply:
ok, thx.

Reply:
we will fix it.